packages/scheduler/src/forks/SchedulerFeatureFlags.www.js

```diff
@@ -7,13 +7,17 @@
  * @flow
  */
 
-const {enableProfiling: enableProfilingFeatureFlag} =
+const {
+  enableProfiling: enableProfilingFeatureFlag,
+  enableSchedulerYield: enableSchedulerYieldFeatureFlag,
+} =
   // $FlowFixMe[cannot-resolve-module]
   require('SchedulerFeatureFlags');
 
 export const enableSchedulerDebugging = true;
 export const enableProfiling: boolean =
   __PROFILE__ && enableProfilingFeatureFlag;
+export const enableSchedulerYield = enableSchedulerYieldFeatureFlag;
```

Comment:
i had overlooked updating this before, which is (apparently) what broke the build with these changes

Reply:
Is Meta actually running the postTask build anywhere? I don't think there's much benefit to switching to postTask unless we can also use yield so maybe we can skip this feature flag and just bundle the postTask + yield changes together.

Reply:
no, we're not - that's a good point.

the only justification for the flag i can think of is that A/Bing the PostTask scheduler with and without `yield` might provide better signal for giving feedback to the chrome team for the origin trial?

Reply:
Personally I'd rather just kill any flag that we don't have a solid reason for, it's already too complicated. But I'll leave it up to you.